### PR TITLE
Fix some minor issues

### PR DESCRIPTION
**Pull request by leofeyer**
… such as typos, missing phpDoc comments, leftover Symfony 4.4 workarounds and the like.

### CODE REVIEW

File: core-bundle/src/Controller/InsertTagsController.php [hunk 1]
```diff
@@ -26,20 +25,16 @@
...
     }
 
     public function renderAction(Request $request, string $insertTag): Response
     {
-        $this->framework->initialize();
-
-        $it = $this->framework->createInstance(InsertTags::class);
-
-        $response = new Response($it->replace($insertTag, false));
+        $response = new Response($this->insertTagParser->replaceInline($insertTag));
```
Comment (leofeyer) [inline core-bundle/src/Controller/InsertTagsController.php:37]: @ausi Just double checking: If `$blnCache` is `false`, we have to use the `replaceInline()` method, right?

Comment (ausi) [reply core-bundle/src/Controller/InsertTagsController.php:37]: ✅

**Review by ausi — DISMISSED** [review]

File: installation-bundle/src/Database/Installer.php [hunk 2]
```diff
@@ -110,7 +111,7 @@ public function compileCommands(): void
         $fromSchema = $this->connection->createSchemaManager()->createSchema();
         $toSchema = $this->schemaProvider->createSchema();
 
-        $diff = $fromSchema->getMigrateToSql($toSchema, $this->connection->getDatabasePlatform());
+        $diff = (new Comparator())->compareSchemas($fromSchema, $toSchema)->toSql($this->connection->getDatabasePlatform());
```
Comment (Toflar) [inline installation-bundle/src/Database/Installer.php]: This should take `createComparator()` on the platform if DBAL 3.x in order to fix the wrong lenght comparisons.

**Review by Toflar — CHANGES_REQUESTED** [review]

File: installation-bundle/src/Database/Installer.php (hunk 2 above)
Comment (leofeyer) [reply installation-bundle/src/Database/Installer.php]: Changed in 39895e4c1ad86ceedc986e8b559c0d9a186de700.

**Review by Toflar — DISMISSED** [review]
Thanks!

**Pull request merged**